### PR TITLE
Working laptops

### PR DESCRIPTION
**Pull request by RarkGrames**
#### Summary
```SUMMARY: Content "Replaced the broken laptop with a working one"```

#### Purpose of change
All the in-game laptops are broken and only can be used for components. There are multiple recipes that require a laptop, so replacing the item with a working one is reasonable.

#### Describe the solution
- Changing the `broken laptop` item into a tool
- Changing the disassembly recipe

#### Testing
Spawned the item, checked out the `use` actions.

### CODE REVIEW

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -2092,11 +2092,12 @@
...
       [ [ "processor", 1 ] ],
       [ [ "RAM", 4 ] ],
+      [ [ "circuit", 1 ] ],
       [ [ "cable", 6 ] ],
       [ [ "amplifier", 1 ] ],
+      [ [ "power_supply", 1 ] ],
       [ [ "plastic_chunk", 6 ] ],
-      [ [ "small_lcd_screen", 1 ] ],
-      [ [ "small_storage_battery", 2 ] ]
+      [ [ "large_lcd_screen", 1 ] ]
```
Comment (Moumix3) [inline data/json/recipes/recipe_deconstruction.json]: The laptop is 2.7kg and has a volume of 2.5L, and the "large_lcd_screen" is an item that weighs 13kg, and has a volume of 15L, so it doesn't make sense to get that from disassembling it. I would revert back to a small_lcd_screen (0.5kg, 0.25L).

**Comment by Maleclypse** [comment]
Have you considered adding music, hand held game etc iuse actions? No stress if that's not a direction you want to go.

**Comment by RarkGrames** [comment]
They are already in there with  `EINKTABLETPC `and `PORTABLE_GAME`.

**Comment by Maleclypse** [comment]
> They are already in there with `EINKTABLETPC `and `PORTABLE_GAME`.

I missed those despite specifically looking for things that I thought would say those words.  Thanks!

**Pull request merged**